sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -143,6 +206,23 @@ def __init__(
         self.documents_cancelled_count = kwargs.get('documents_cancelled_count', None)
         self.total_characters_charged = kwargs.get('total_characters_charged', None)
 
+    @classmethod
+    def _from_generated(cls, batch_status_details):
+        return cls(
+            id = batch_status_details.id,
+            created_on = batch_status_details.created_date_time_utc,
+            last_updated_on = batch_status_details.last_action_date_time_utc,
+            status = batch_status_details.status,
+            error = DocumentTranslationError._from_generated(batch_status_details.error),
```

Comment:
`error` is not a required parameter so the service might not always return it. I would add a conditional either here or in the helper method in the case that `batch_status_details.error` == None so we don't get an AttributeError when trying to wrap the model 
edit: same comment for DocumentStatusDetail.error